docs/tutorials/index.md

```diff
@@ -6,37 +6,22 @@ seo:
 
 The XRP Ledger tutorials walk you through the steps to learn and get started with the XRP Ledger and to use the ledger for advanced use cases.
 
-## Get Started with SDKs
+## Get Started with Your Favorite Programming Language
```

Comment:
This change affects the in-page nav on the right. It may be better to keep the heading as-is and include "your favorite programming language" in the paragraph.